scio-examples/src/main/scala/com/spotify/scio/examples/extra/AvroExample.scala

```diff
@@ -24,32 +24,17 @@
 package com.spotify.scio.examples.extra
 
 import com.spotify.scio._
-import com.spotify.scio.coders.Coder
 import com.spotify.scio.avro._
-import com.spotify.scio.avro.Account
-import com.spotify.scio.avro.types.AvroType
+import com.spotify.scio.avro.{Account => JAccount}
+import com.spotify.scio.coders.Coder
 import com.spotify.scio.io.ClosedTap
 import org.apache.avro.Schema
 import org.apache.avro.generic.{GenericData, GenericRecord}
 
 import scala.jdk.CollectionConverters._
 
 object AvroExample {
-  @AvroType.fromSchema("""{
```

Comment:
There is a problem with removing this. We need to substitute this with alternative or plan some migration. It is straightforward to port pure typed API, but everything related to schemas is not straightforward, but it uses the old Scio's macros